### DISCUSSION TITLE
Any easy Next.js deployment solution?

### DISCUSSION THREAD

**Discussion by rivierejordan** [drknzz/GitHub-Achievements#413] [Q&A] [answered]
Does someone know any Next.js deployment solution that are easy to use?

**Comment by Boutzi** [comment, marked as answer]
Sure !!!! There is Vercel, the easiest solution for it ! They actualy made Next.js.
If you want a more complex way to deploy Next.js apps in the future, consider using AWS Amplify !

**Reply by rivierejordan** [reply]
Thanks a lot

**Comment by SatyamPote** [comment]
answer  me is all right
